package.json

```diff
@@ -40,10 +40,21 @@
     "release": "npm run build && npm publish ./build --access=public",
     "build:watch": "tsc -w -p tsconfig.release.json",
     "lint": "eslint . --ext .ts,.tsx",
-    "test": "jest --coverage",
+    "test": " jest --coverage --coverageReporters='json-summary'",
+    "make-badges": "coverage-badges --output ./coverage.svg",
```

Comment:
Again, my fault. I wanted to run both test and creating coverage badge in the same script and somehow, missed implementing in this repo. Could you please remove `make-badges` script and add it to `test script` like `"test": "jest --coverage && coverage-badges --output ./coverage.svg"`